lib/application.js

```diff
@@ -80,36 +87,46 @@ class Application extends events.EventEmitter {
     }
   }
 
-  async loadScript(place, fileName) {
-    const { name, ext } = path.parse(fileName);
-    if (ext !== '.js' || name.startsWith('.')) return;
+  async loadDomain(fileName) {
+    const rel = fileName.substring(this.domainPath.length + 1);
+    const [name, ext] = rel.split('.');
```

Comment:
This will fail for names with dots in it which is not that uncommon (i.e. `'name.with.dots.js'`).

Reply:
You are right, I'll rewrite this